src/providers/ColorSchemeProvider.tsx

```diff
@@ -16,19 +15,15 @@ const ColorSchemeProvider = ({ children }: ColorSchemeProps) => {
     getInitialValueInEffect: true,
   });
 
+  // const current = localStorage
```

Comment:
@devjoylee You can delete these codes (L18~L20) if you're not going to use in the future:good:

Reply:
I was trying to fix the rendering bug but I couldn't do it, so I commented it out. I'll keep it as is for now, probably I or somebody else get some idea in the future..?